backend/scripts/db_sample.sql

```diff
@@ -11,6 +11,7 @@ CREATE TABLE RSS_LIST (
 DROP TABLE IF EXISTS RSS_HISTORY CASCADE;
 CREATE TABLE RSS_HISTORY (
     id serial PRIMARY KEY,
+    rss_id BIGINT NOT NULL,
```

Comment:
```suggestion
    rss_list_id BIGINT NOT NULL,
```
Should be `rss_list_id` (see https://github.com/Yoann-Abbes/The-count-of-money/pull/22)